packages/@sanity/cli/test/alias.test.ts

```diff
@@ -0,0 +1,96 @@
+import {describeCliTest, testConcurrent} from './shared/describe'
+import {getTestRunArgs, runSanityCmdCommand, studioVersions} from './shared/environment'
+
+describeCliTest('CLI: `sanity dataset alias`', () => {
+  describe.each(studioVersions)('%s', (version) => {
+    jest.setTimeout(30 * 1000)
```

Comment:
Is this the way of defining timeout which is buggy in Jest?

Reply:
No, it's the `testTimeout` set in `jest.config.cjs` which isn't working when using jest projects - using `jest.setTimeout` still works, but is tedious when you have to define it for every single test. This one can be removed since it is the default, however - will fix.

Reply:
OK, I see 🙏